src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -703,6 +703,19 @@ class ResourceServiceSpec extends FlatSpec with Matchers with ScalaFutures with
     policyDAO.listAccessPolicies(resource).unsafeRunSync() should not be empty
   }
 
+  it should "allow for auth domain groups on a deleted resource to be deleted" in {
+    val resourceType = constrainableResourceType.copy(reuseIds = false)
+    val managedGroupName = "fooGroup"
+    constrainableService.createResourceType(resourceType).unsafeRunSync()
+    constrainableService.createResourceType(managedGroupResourceType).unsafeRunSync()
+    managedGroupService.createManagedGroup(ResourceId(managedGroupName), dummyUserInfo).unsafeRunSync()
+    val resource = constrainableService.createResource(resourceType, ResourceId(UUID.randomUUID().toString), Map(AccessPolicyName(constrainableReaderRoleName.value) -> constrainablePolicyMembership), Set(WorkbenchGroupName(managedGroupName)), dummyUserInfo.userId).unsafeRunSync()
+
+    runAndWait(constrainableService.deleteResource(resource.fullyQualifiedId))
+    runAndWait(managedGroupService.deleteManagedGroup(ResourceId(managedGroupName)))
+    managedGroupService.loadManagedGroup(ResourceId(managedGroupName)).unsafeRunSync() shouldBe None
+  }
```

Comment:
just being a little paranoid here, but can you add to your test another resource with an auth domain and verify that it remains unchanged? The failure case I am thinking about on the postgres side is that the query is wrong and deletes all auth domains.